design/nodes-machines-and-hosts.md

```diff
@@ -57,7 +57,7 @@ such as draining the node for maintenance. The definition of Node is
 owned by the Kubernetes core, and offers few opportunities to attach
 custom data.
 
-[Machine](https://kubernetes-sigs.github.io/cluster-api/common_code/machine_controller.html)
+[Machine](https://cluster-api.sigs.k8s.io/developer/architecture/controllers/machine)
```

Comment:
It should be https://cluster-api.sigs.k8s.io/developer/core/controllers/machine

Reply:
Made this change in the latest commit